examples/tv-casting-app/linux/simple-app.cpp

```diff
@@ -76,6 +78,30 @@ CHIP_ERROR InitCommissionableDataProvider(LinuxCommissionableDataProvider & prov
                          options.payload.discriminator.GetLongValue());
 }
 
+class DiscoveryDelegateImpl : public DiscoveryDelegate
+{
+private:
+    int commissionersCount = 0;
+
+public:
+    void HandleOnAdded(Strong<CastingPlayer> player) override
+    {
+        if (commissionersCount == 0)
+        {
+            ChipLogProgress(AppServer, "Select discovered commissioner to request commissioning");
+
+            ChipLogProgress(AppServer, "Example: cast request 0");
+        }
+        ++commissionersCount;
+        ChipLogProgress(AppServer, "Discovered Commissioners #%d", commissionersCount);
```

Comment:
Is the commissioner number appearing correctly or is it still skipping by one?

Reply:
Appearing correctly, it was caused by ChipLogProgress